.github/workflows/r-ci.yml

```diff
@@ -37,12 +37,12 @@ jobs:
       - name: MkVars
         run: mkdir ~/.R && echo "CXX17FLAGS=-Wno-deprecated-declarations -Wno-deprecated" > ~/.R/Makevars
 
-      - name: Build and install libtiledbsoma
-        run: sudo scripts/bld --prefix=/usr/local 
+      #- name: Build and install libtiledbsoma
+      #  run: sudo scripts/bld --prefix=/usr/local 
 
-      - name: Call ldconfig
-        if: ${{ matrix.os == 'ubuntu-latest' }}
-        run: sudo ldconfig
+      #- name: Call ldconfig
+      #  if: ${{ matrix.os == 'ubuntu-latest' }}
+      #  run: sudo ldconfig
```

Comment:
Remove?

Reply:
At first I left it because we may want it in a matrix to try the other mode.  I then realized that the existing R/Python interop test has this covered.  So we really could remove if it bugs you greatly.  It is a placeholder of things we do not do in other places, and sometimes keeping things around commented-out can be beneficial.

Reply:
It doesn't bug me either way. I was just wondering if this was left commented out intentionally or if you meant to delete it.

Reply:
It was mostly a post-it for my future self.  I tend to remove old comments, and often ask for it too.  But this CI yaml files as some things others don't do so I kept.  There is of course `git` histore by then again `grep` (and `ag`) are such good friends of mine...

So thanks for tolerating it here :wink: